wfc/src/main/java/com/webfuzzing/commons/faults/FaultCategory.java

```diff
@@ -72,4 +81,14 @@ public enum FaultCategory {
     public String getLabel() {
         return "F" + code + ":" + name;
     }
+
+    /**
+     * A camelCase label for this category, to be used in test case naming.
+     * This will enable tests to have more descriptive names and easy detection of tests with faults
+     *
+     * @return the test case label
+     */
+    public String getTestCaseLabel() {
```

Comment:
as this is an enumeration, and the fields are public final, there is no need for a direct getter